program/rust/src/accounts/score.rs

```diff
@@ -0,0 +1,158 @@
+use {
+    super::{
+        AccountHeader,
+        PythAccount,
+    },
+    crate::c_oracle_header::{
+        PC_ACCTYPE_SCORE,
+        PC_MAX_PUBLISHERS,
+        PC_MAX_SYMBOLS,
+    },
+    bytemuck::{
+        Pod,
+        Zeroable,
+    },
+    solana_program::{
+        program_error::ProgramError,
+        pubkey::Pubkey,
+    },
+    std::cmp::max,
+};
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct PublisherScoresAccount {
+    pub header:                AccountHeader,
+    pub num_publishers:        usize,
+    pub num_symbols:           usize,
+    // a constant used to normalize the scores
+    pub z:                     u32,
+    pub publisher_permissions: [[bool; PC_MAX_SYMBOLS as usize]; PC_MAX_PUBLISHERS as usize],
```

Comment:
`[bool; usize]` is like C++ then it'll take usize bytes, i recommend using u64 (or word size) and accessing it (or external library if program size allows it)

Reply:
Yeah I guessed it might be the same. Fixed. Thank you.